docs/en/observability/ingest-logs.asciidoc

```diff
@@ -0,0 +1,188 @@
+
+[NOTE]
+=====
+If you haven't already, you need to install {es} for storing and searching your data, and {kib} for visualizing and
+managing it. For more information, see <<install-observability,Get started>>.
+=====
+
+[[install-filebeat]]
+== Step 1: Install {beatname_uc}
+
+Install {beatname_uc} on all the servers you want to monitor.
+
+To download and install {beatname_uc}, use the commands that work with your system:
+
+include::{beats-repo-dir}/tab-widgets/install-widget.asciidoc[]
+
+[[other-filebeat-installations]]
+=== Other installation options
+
+* {filebeat-ref}/running-on-kubernetes.html[Kubernetes]
+* {filebeat-ref}/running-on-docker.html[Docker]
+* {filebeat-ref}/running-on-cloudfoundry.html[Cloud Foundry]
+* {filebeat-ref}/setup-repositories.html[APT or YUM]
+* https://www.elastic.co/downloads/beats/filebeat[Download page]
+
+[[logs-connect-to-stack]]
+== Step 2: Connect to {es} and {kib}
+
+Connections to {es} and {kib} are required to set up Filebeat.
+
+Set the connection information in `filebeat.yml`. To locate this configuration file,
+see {filebeat-ref}/directory-layout.html[Directory layout].
+
+include::{beats-repo-dir}/tab-widgets/set-connection-widget.asciidoc[]
+
+To learn more about required roles and privileges, see {filebeat-ref}/feature-roles.html[Grant users access to secured resources].
+
+[NOTE]
+=====
+
+You can send data to other {filebeat-ref}/configuring-output.html[outputs], such as Logstash, 
+but that requires additional configuration and setup.
+
+=====
+
+[[enable-logs-integrations]]
+== Step 3: Enable and configure integrations
+
+{beatname_uc} uses integrations to collect and parse log data.
+
+. To identify the integrations you need to enable, run:
++
+--
+include::{beats-repo-dir}/tab-widgets/list-modules-widget.asciidoc[]
+--
++
+Can’t find an integration for your file type? Skip this section and
+{filebeat-ref}/configuration-filebeat-options.html[configure the input manually].
+
+. From the installation directory, enable one or more integrations. For example, the
+following command enables the `system`, `nginx`, and `mysql` integration
+configs:
++
+--
+include::{beats-repo-dir}/tab-widgets/enable-modules-widget.asciidoc[]
+--
+
+. In the module configs under `modules.d`, change the module settings to match
+your environment.
++
+For example, log locations are set based on the OS. If your logs aren't in
+default locations, set the `paths` variable:
++
+--
+[source,yaml]
+----
+- module: nginx
+  access:
+    var.paths: ["/var/log/nginx/access.log*"] <1> 
+----
+--
+
+To see the full list of variables for an integration, see the documentation under
+{filebeat-ref}/filebeat-modules.html[Modules].
+
+[TIP]
+=====
+To test your configuration file, change to the directory where the Filebeat binary
+is installed, and run Filebeat in the foreground with the following options specified: 
+./filebeat test config -e. Make sure your config files are in the path expected by 
```

Comment:
Wrap commands and command line options in back ticks to apply monospace formatting.